### PR TITLE
ARTEMIS-4785 Add log4j2-default.properties for non-run CLI commands

### PR DESCRIPTION
**Pull request by brusdev**
The run command uses log4j2.properties and all other commands use log4j2-default.properties.

### CODE REVIEW

File: artemis-cli/src/main/java/org/apache/activemq/artemis/cli/Shell.java [hunk 1]
```diff
@@ -122,7 +125,22 @@ public static void runShell(boolean printBanner) {
...
+                     boolean IS_WINDOWS = System.getProperty("os.name").toLowerCase().trim().startsWith("win");
+                     if (IS_WINDOWS) {
+                        command.add("cmd");
+                        command.add("/c");
+                        command.add(Paths.get(artemisInstance, "artemis.cmd").toAbsolutePath().toString());
+                     } else {
+                        command.add(Paths.get(artemisInstance, "bin", "artemis").toAbsolutePath().toString());
+                     }
+                     command.addAll(parser.parse(line, 0).words());
+                     new ProcessBuilder(command).inheritIO().start().waitFor();
```
[the remark this answers is not in the document]
Comment (brusdev) [reply artemis-cli/src/main/java/org/apache/activemq/artemis/cli/Shell.java:140]: I'll remove this part.

File: artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/Create.java [hunk 2]
```diff
@@ -77,6 +77,7 @@ public class Create extends InstallAbstract {
    public static final String BIN_ARTEMIS_SERVICE = "bin/" + ARTEMIS_SERVICE;
    public static final String ETC_ARTEMIS_PROFILE = "artemis.profile";
    public static final String ETC_LOG4J2_PROPERTIES = "log4j2.properties";
+   public static final String ETC_LOG4J2_DEFAULT_PROPERTIES = "log4j2-default.properties";
```
[the remark this answers is not in the document]
Comment (brusdev) [reply artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/Create.java:80]: I'm not sure to get your point. My point is that only the run command uses log4j2.properties all other commands use log4j2-default.properties.

File: artemis-cli/src/main/resources/org/apache/activemq/artemis/cli/commands/etc/artemis.profile [hunk 3]
```diff
@@ -60,4 +60,6 @@ if [ "$1" = "run" ]; then :
 
     # Uncomment for async profiler
     # DEBUG_ARGS="-XX:+UnlockDiagnosticVMOptions -XX:+DebugNonSafepoints"
+else
+    JAVA_ARGS="$JAVA_ARGS -Dlog4j2.configurationFile=${ARTEMIS_INSTANCE_ETC_URI}log4j2-default.properties "
```
[the remark this answers is not in the document]
Comment (brusdev) [reply artemis-cli/src/main/resources/org/apache/activemq/artemis/cli/commands/etc/artemis.profile:64]: Good point, I'll fix it.

File: artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/Create.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (gemmellr) [reply artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/Create.java:80]: My point is that it feels like 'log4j2-default' isnt a very clear descriptive name for everyone else and could easily be misunderstood. I know that we will understand what its for, but it seems like something others could misinterpret, i.e those who havent seen the Jira/this discussion/this PR to understand exactly why that file is being created and what that file is not used for. Especially when in actual Log4J2 terms the existing log4j2.properties is actually 'the default file' (to Log4J it is the highest preendence non-test configuration file, and what will get used if you dont specify -Dlog4j2.configurationFile=foo). Something more descriptive of its usage like e.g log4j2-tools/utilities/foo.properties makes more explicit what the file is being aimed at than just a '-default' suffix.

[the remark this answers is not in the document]
Comment (brusdev) [reply artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/Create.java:80]: Now I see your point, I'll use the `utility` suffix.

**Comment by brusdev** [comment]
Replaced by #5136

**Pull request closed without merging**